Added marketplace contribution template and updated readme (#165)
mia-platform/community: .github/ISSUE_TEMPLATE/marketplace-contribution.md

---
name: Marketplace contribution

> about: Add a new component to Mia Platform marketplace

Reviewer: ```suggestion
about: Add a new component to Mia-Platform marketplace
```

> title: ''

Reviewer: ```suggestion
title: 'Add new marketplace item '
```

> labels: enhancement

Reviewer: ```suggestion
labels: marketplace
```

assignees: ''
---

Thanks for contributing to Mia-Platform Marketplace!

If you wish to request the addition of a new component, please fill in the necessary fields below.  
When the issue will be opened, a Mia-Platform referent will contact you to verify and manage the component addition.

Here, the required information to be fulfilled:

- **name**: write here the component name appearing in the Marketplace card

- **description**: write here a brief description (10 to 20 words) regarding the service functionalities

> - **type**: write here the type of your component; it could be: plugin, template, example, application, or proxy.  

Reviewer: ```suggestion
- **type**: write here the type of your component; it could be: `plugin`, `template`, `example`, `application`, or `proxy`.  
```

Visit this link to know in which type of service the component may be included - <https://docs.mia-platform.eu/docs/marketplace/add_to_marketplace/contributing_overview>

- **supportedByImage**: attach the image that will be associated with the Company that has produced it

- **supportedBy**: write here a label to identify the Company that has produced the component (only used if supportedByImage is not provided)

- **image**: attach the image that will be associated to the component

- **repository URL**: write here the link to reach out the component repository

> In the repository, please make sure to provide us a detailed ReadMe.

Reviewer: ```suggestion
In the repository, please make sure to provide a detailed README.
```

why *us*?

> The optimal solution would be to have two ReadMes that essentially provide two types of information:

Reviewer: ```suggestion
The optimal solution would be to have two README files that essentially provide two types of information:
```

 1. why should a user use your example/template? what technology stack does it use, etc.?
 2. once installed, what must the developer do to use the repository? (e.g. are there specific commands to run? resources to install to prepare the environment? etc.)

In case your component is a **plugin**:
write here its docker image

In case your component is a **template/example**:

> prepare and write here the required archiveUrl; check here a format example for this required resource: <https://github.com/mia-platform-marketplace/Node.js-Hello-World-Microservice-Example/archive/master.tar.gz>

Reviewer: markdown link

When a Mia-Platform referent has contacted you subsequently to the publication of this issue, you can manage together the correct configuration of the component so that it can be configurable and correctly deployable from Mia-Platform Console.

> If you would like to learn more about which are the expected properties of a Mia-Platform Marketplace component, depending on the type of component, please read this documentation page carefully: <https://docs.mia-platform.eu/docs/marketplace/add_to_marketplace/contributing_overview#how-to-configure-a-new-component>

Reviewer: markdown link
